src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const longTermDiskont = 50;
```

Comment:
There is a typo in the variable name 'longTermDiskont'. It should be 'longTermDiscount' to accurately reflect its purpose and to be grammatically correct.